target/start-mailserver.sh

```diff
@@ -95,7 +97,7 @@ function register_functions() {
 
 	################### >> fix funcs
 
-	_register_fix_function "_fix_var_mail_permissions"
+#	_register_fix_function "_fix_var_mail_permissions"
```

Comment:
The current function is changing the owner of all mailboxes to 5000:5000 (docker user). 
But while running the server, the mailboxes will be created with userid and group id (not with 5000:5000) with rights 7000. 
When the container restarts , the owner of mailboxes are changed to 5000 and mailboxes won't be accessible. 

Im trying to fix the owner of /var/mail when the container is initialized (in DokerFile) And letthe mailboxe owner as they have been created.